intro_to_r/intro_to_r.Rmd

```diff
@@ -66,7 +82,7 @@ boys to newborn girls, so he gathered the baptism records for children born in
 London for every year from 1629 to 1710.  We can take a look at the data by 
 typing its name into the console.
 
-```{r view-data, eval=FALSE}
+```{r view-data}
 arbuthnot
```

Comment:
You can still do `rm(x)` to get back to a clean slate, but I'm not sure what I'd teach here. I'd probably lean more towards teaching people how to restart R from inside Rstudio (and focus on keeping track of everything you're doing in the editor), but that's probably too much to cover on day 1.